conftest.py

```diff
@@ -0,0 +1,9 @@
+# List of files that pytest should ignore
+
+collect_ignore = ["setup.py",
+                  "skimage/io/_plugins"]
+try:
+    import visvis
```

Comment:
Wrong link above, should be https://github.com/scikit-image/scikit-image/blob/5d0352f6225e709fd9dceed28b98b096b2d84125/skimage/measure/mc_meta/visual_test.py.